R/insertPackage.R

```diff
@@ -159,10 +159,20 @@ insertPackage <- function(file,
     invisible(NULL)
 }
 
-
+##' @rdname insertPackage
+insertPackages <- function(files, ...){
+    lapply(files, insertPackage, ...)
+}
 
 ##' @rdname insertPackage
-insert <- function(...) insertPackage(...)
+insert <- function(...) {
+    args <- list(...)
+    if("files" %in% names(args)){
+        insertPackages(...)
```

Comment:
how named `files` argument to `...` is meant to work here? are there any examples or documentation?

Reply:
See line 31 in this file

Reply:
I think providing vector input to `file` argument is fine too, shouldn't need extra `files` argument handling

Reply:
fixed in 207fa40